### PR TITLE
Client auth

### PR DESCRIPTION
**Pull request by enjeck**
https://trac.torproject.org/projects/tor/ticket/31069

### CODE REVIEW

**Comment by emmapeel2** [comment]
The contents are very good!

But could you please name the folders as the slug of the article? so the URL makes more sense, and is better for CEO, and for working etc...

**Comment by enjeck** [comment]
@emmapeel2 Edited :) I was worried about the slugs being too long

File: content/onionservices/onion-service-asking-for-private-key/contents.lr [hunk 1]
```diff
@@ -0,0 +1,9 @@
+_model: question
+---
+title: Why is X.onion asking for a private key?
+---
+seo_slug: onion-service-asking-for-private-key
+---
+description:
+If you see this, then you're trying to access an authenticated onion. An authenticated onion service is an onion service that requires you to provide an authentication token (in this case, a private key) before accessing the service.
+[Learn more about onion authentication.](https://tb-manual.torproject.org/onion-services/#client-auth)
```
Comment (asn-d6) [inline content/onionservices/onion-service-asking-for-private-key/contents.lr:9]: The way this paragraph reads, people might be confused that the private key is provided (i.e. transmitted) to the service. Perhaps it's worth clarifying in a sentence that "The private key is not transmitted to the service, and it's only used to decrypt its descriptor locally", or a variant of this.

Comment (gusgustavo) [reply content/onionservices/onion-service-asking-for-private-key/contents.lr:9]: Fixed!

**Comment by gusgustavo** [comment]
I merged all questions to just one, so it's easy to an user to click and learn. Thanks Cleo!

**Pull request closed without merging**